packages/gatsby-plugin-image/package.json

```diff
@@ -79,6 +79,12 @@
     "@babel/parser": "^7.12.5",
     "@babel/traverse": "^7.12.5",
     "babel-jsx-utils": "^1.0.1",
+    "@babel/plugin-syntax-jsx": "^7.12.1",
+    "@babel/plugin-syntax-typescript": "^7.12.1",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "jscodeshift": "^0.11.0",
```

Comment:
Does this have to be a dependency as well as asking users to add it globally? It's a pretty chunky dependency to make all user sites doenload

Reply:
It did at the time it was included but since we're no longer using anything from the API it shouldn't be necessary. I'll double check.

Reply:
It has to be a devDependency for testing. But have moved it there.